resources/lang/en/community.php

```diff
@@ -23,13 +23,13 @@
         'header' => [
             // size in font-size
             'big_description' => 'Love osu!?<br/>
-                                Support the osu! dev :D',
+                                Support the osu!dev :D',
```

Comment:
should probably just be "Support the development of osu! :D" or something similar.